static/env-vars/activitylog_configvars.md

```diff
@@ -2,10 +2,6 @@ Environment variables for the **activitylog** service
 
 | Name | Introduction Version | Type | Description | Default Value |
 |---|---|---|---|:---|
-|`OC_TRACING_ENABLED`<br/>`ACTIVITYLOG_TRACING_ENABLED`| 1.0.0 |bool|`Activates tracing.`|`false`|
```

Comment:
@dragonchaser from my pov, they should appear in the deletion section.

Reply:
Let us put that on the deleted list with a explanation why we did this without deprecation.